storage/atsd/converter_test.go

```diff
@@ -0,0 +1,596 @@
+						IoTime:          123,
+						WeightedIoTime:  123,
+					},
+				},
+				TaskStats: info.LoadStats{
+					NrSleeping:        123,
+					NrRunning:         123,
+					NrStopped:         123,
+					NrUninterruptible: 123,
+					NrIoWait:          123,
+				},
+			},
+			seriesCommands: []*atsdNet.SeriesCommand{
+				atsdNet.NewSeriesCommand(entityName, containerCpuUsageUser, atsdNet.Uint64(123)).
+					SetMetricValue(containerCpuUsageTotal, atsdNet.Uint64(123)).
+					SetMetricValue(containerCpuUsageSystem, atsdNet.Uint64(123)).
+					SetMetricValue(containerCpuLoadAverage, atsdNet.Uint64(123)).
+					SetMetricValue(containerCpuLoadAverage, atsdNet.Uint64(123)).
+					SetTimestamp(atsdNet.Millis(123456789)),
+				atsdNet.NewSeriesCommand(entityName, containerMemoryWorkingSet, atsdNet.Uint64(123)).
+					SetMetricValue(containerMemoryUsage, atsdNet.Uint64(123)).
+					SetMetricValue(containerMemoryCache, atsdNet.Uint64(123)).
+					SetMetricValue(containerMemoryRSS, atsdNet.Uint64(123)).
+					SetMetricValue(containerMemoryHierarchicalDataPgfault, atsdNet.Uint64(123)).
+					SetMetricValue(containerMemoryHierarchicalDataPgmajfault, atsdNet.Uint64(123)).
+					SetMetricValue(containerMemoryContainerDataPgfault, atsdNet.Uint64(123)).
+					SetMetricValue(containerMemoryContainerDataPgmajfault, atsdNet.Uint64(123)).
+					SetMetricValue(containerMemoryFailcnt, atsdNet.Uint64(123)).
+					SetTimestamp(atsdNet.Millis(123456789)),
+				atsdNet.NewSeriesCommand(entityName, containerTaskStatsNrIoWait, atsdNet.Uint64(123)).
+					SetMetricValue(containerTaskStatsNrRunning, atsdNet.Uint64(123)).
+					SetMetricValue(containerTaskStatsNrSleeping, atsdNet.Uint64(123)).
+					SetMetricValue(containerTaskStatsNrStopped, atsdNet.Uint64(123)).
+					SetMetricValue(containerTaskStatsNrUninterruptible, atsdNet.Uint64(123)).
+					SetTimestamp(atsdNet.Millis(123456789)),
+				atsdNet.NewSeriesCommand(entityName, containerNetworkRxBytes, atsdNet.Uint64(123)).
+					SetMetricValue(containerNetworkRxDropped, atsdNet.Uint64(123)).
+					SetMetricValue(containerNetworkRxErrors, atsdNet.Uint64(123)).
+					SetMetricValue(containerNetworkRxPackets, atsdNet.Uint64(123)).
+					SetMetricValue(containerNetworkTxBytes, atsdNet.Uint64(123)).
+					SetMetricValue(containerNetworkTxDropped, atsdNet.Uint64(123)).
+					SetMetricValue(containerNetworkTxErrors, atsdNet.Uint64(123)).
+					SetMetricValue(containerNetworkTxPackets, atsdNet.Uint64(123)).
+					SetMetricValue(containerNetworkTcpStatEstablished, atsdNet.Uint64(123)).
+					SetMetricValue(containerNetworkTcpStatSynSent, atsdNet.Uint64(123)).
+					SetMetricValue(containerNetworkTcpStatSynRecv, atsdNet.Uint64(123)).
+					SetMetricValue(containerNetworkTcpStatFinWait1, atsdNet.Uint64(123)).
+					SetMetricValue(containerNetworkTcpStatFinWait2, atsdNet.Uint64(123)).
+					SetMetricValue(containerNetworkTcpStatTimeWait, atsdNet.Uint64(123)).
+					SetMetricValue(containerNetworkTcpStatClose, atsdNet.Uint64(123)).
```

Comment:
By using the same value for every field you're not really testing much other than that each field is set.